homeassistant/components/airnow/sensor.py

```diff
@@ -78,6 +84,10 @@ def station_extra_attrs(data: dict[str, Any]) -> dict[str, Any]:
         extra_state_attributes_fn=lambda data: {
             ATTR_DESCR: data[ATTR_API_AQI_DESCRIPTION],
             ATTR_LEVEL: data[ATTR_API_AQI_LEVEL],
+            ATTR_TIME: datetime.strptime(
+                f"{data[ATTR_API_REPORT_DATE]} {data[ATTR_API_REPORT_HOUR]}",
+                "%Y-%m-%d %H",
+            ).replace(tzinfo=get_time_zone(data[ATTR_API_REPORT_TZ])),
```

Comment:
```suggestion
            ATTR_TIME: datetime.strptime(
                f"{data[ATTR_API_REPORT_DATE]} {data[ATTR_API_REPORT_HOUR]}",
                "%Y-%m-%d %H",
            ).replace(tzinfo=get_time_zone(data[ATTR_API_REPORT_TZ])).isoformat(),
```

Reply:
We should return the datetime as an iso-formatted string in the attributes

Reply:
I've made the requested change, but out of curiosity—I think I saw `datetime` objects returned elsewhere in `extra_state_attributes`. Is there a reason to prefer converting it here over (what I assume) having it automatically converted/handled later?

Reply:
For attributes it's not handled later so therefore we need to make sure we store it properly here.